runtime/Cargo.toml

```diff
@@ -211,9 +211,7 @@ version = '0.0.1'
 [dependencies.peaq-pallet-storage]
 git = 'https://github.com/peaqnetwork/peaq-storage-pallet.git'
 default-features = false
-branch = 'dev'
-version = '0.1.0'
-
+branch = 'features/1203099733411537_add-weights-after-benchmarking'
```

Comment:
Please change back after the `features/1203099733411537_add-weights-after-benchmarking` is merged.

Reply:
Yes, I have that point in my mind